addons/test_mail/tests/test_mail_composer.py

```diff
@@ -3264,6 +3264,25 @@ def test_mail_composer_wtpl_reply_to_force_new(self):
                                 },
                                )
 
+    @users('employee')
+    def test_mail_sending_on_non_thread_model(self):
```

Comment:
There is a "TestNoThread" class in test_mail_thread_internals holding some tests for "non thread" models :)